executor/services/api_client.py

```
@@ -0,0 +1,404 @@
+            # Download skill ZIP
+            download_path = f"/api/v1/kinds/skills/{skill_id}/download?namespace={skill_namespace}"
+            response = self.client.get(download_path, timeout=self.DOWNLOAD_TIMEOUT)
+
+            if not response:
+                logger.error(f"[SkillDownloader] Failed to download skill '{skill_name}'")
+                return False
+
+            # Extract ZIP
+            return self._extract_skill_zip(skill_name, response.content)
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)